utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -27,62 +85,78 @@ function Invoke-ResourceRemoval {
         [string] $Type
     )
 
-    switch ($type) {
+    Remove-UnhandledDeleteLock -ResourceId $ResourceId
```

Comment:
For cases where the lock deletion is not discovered using the resource deployment data collection. I experienced this for cases where the deployment and removal failed of the common test.